poll/poll_func.go

```diff
@@ -14,63 +15,85 @@ func PollCmd(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 	err := r.ParseForm()
 	if err != nil {
-		log.Print(err)
+		log.Print("Error: ", err)
 		return
 	}
 	poll, err := NewPollRequest(r.Form)
-	if err != nil {
-		log.Print(err)
-		return
-	}
-	if len(Conf.Token) != 0 && Conf.Token != poll.Token {
-		log.Print("Token missmatch. Check you config.json")
-		return
+	var response_type string
+	var response_text string
+	if err == nil {
+		response_type = "in_channel"
+		response_text = poll.Message + ` #poll`
+	} else {
+		response_type = "ephemeral"
+		response_text = err.Error()
 	}
 
-	c := model.NewClient(Conf.Host)
-	c.TeamId = poll.TeamId
+	var response = `{
```

Comment:
This part looks a little dirty.
Does it had better use [model.CommandResponse type](https://github.com/mattermost/platform/blob/9ee7f661c76d7ef07ac03772a7cf0394217203f3/model/command_response.go#L16)?